packages/strapi-tiptap-editor/admin/src/components/Wysiwyg/index.tsx

```diff
@@ -224,7 +224,7 @@ const WysiwygContent = ({
     injectCSS: false,
     editorProps: {
       attributes: {
-        class: 'utrecht-theme utrecht-theme--media-query-color-scheme utrecht-html utrecht-document',
+        class: 'utrecht-html',
```

Comment:
In plaats van deze class name, kunnen we ook een `<HTMLContent>` React wrapper doen?

Reply:
Met de class-aanpak zullen we alleen invloed hebben op de gerenderde HTML. Maar als we een wrapper met deze `utrecht-html` CSS class gebruiken, kan de class mogelijk ook andere onderdelen van de editor beïnvloeden.